backend/app/organisation/routers.go

```diff
@@ -11,4 +11,10 @@ func OrganisationRouter() {
 	router.POST("/organisations", OrganisationsAllHandler)
 	router.GET("/organisations/{organisation_id}", OrganisationsHandler)
 	router.PATCH("/organisations/{organisation_id}", OrganisationsHandler)
```

Comment:
Maybe it's a typo... are we working with `/{element_id}` or `/:elementID` in endpoints ?

Reply:
With `/:elementID` this is how gin handle path variable.

Reply:
we should then remove the others elements !?

Reply:
Yes.